pkg/builtinchecks/yamls/unsafe-proc-mount.yaml

```diff
@@ -0,0 +1,7 @@
+name: "unsafe-proc-mount"
+description: "Alert on deployments with unsafe /proc mount on containers"
```

Comment:
Some more explanation on what the impact of this is/what it means would be helpful.

Reply:
updated with the impact.